languages/elixir/concepts/pids/links.json

```diff
@@ -1 +1,22 @@
-[]
+[
+  {
+    "url": "https://elixir-lang.org/getting-started/processes.html",
+    "description": "Getting Started guide - Processes"
+  },
+  {
+    "url": "https://samuelmullen.com/articles/elixir-processes-send-and-receive/",
+    "description": "Elixir Processes: Spawn, Send, and Receive"
```

Comment:
PIDs are never mentioned outside of the topic of processes, so I think linking to introductions about processes makes sense here.